docs/uptime-guide/overview.asciidoc

```diff
@@ -1,11 +1,56 @@
 [role="xpack"]
 [[uptime-overview]]
-== Overview
+== Elastic Uptime overview
 
-The Elastic Uptime solution built on the Elastic Stack.
-It allows you to monitor the availability and response time of software services and applications in real time.
+The Elastic Uptime solution is built on the Elastic Stack.
```

Comment:
I know this is from the previous version, but it seems a bit obvious and not worth stating?

Reply:
Sometimes I think it is necessary to state one or two seemingly obvious things up-front to set the context for any readers who aren't familiar with all the concepts, and avoid losing them before you start. But here, I agree this adds nothing, as the Elasticsearch and Kibana are mentioned shortly anyway.